CautionaryAlertsListener.Tests/EventDataFixture.cs

```diff
@@ -0,0 +1,25 @@
+using Hackney.Core.Sns;
+using System.Dynamic;
+
+namespace CautionaryAlertsListener.Tests
+{
+    public static class EventDataFixture
```

Comment:
If this is used only as part of the E2E tests then can we move this to be in the E2E folder